platform/switch/zelda3.ini

```diff
@@ -93,10 +111,10 @@ CancelBirdTravel = 0
 
 [KeyMap]
 # Change what keyboard keys map to the joypad
-# Order: Up, Down, Left, Right, Select, Start, A, B, X, Y, L, R
+Order: Up, Down, Left, Right, Select, Start, A, B, X, Y, L, R

```

Comment:
Why did you remove the #

Reply:
I removed the # to uncomment the keys that actually map to the switch instead of the windows keys that were mapped before. Either way it appears to have no affect as mapping these and then trying any of the menu toggles produced no results.

Reply:
The one you're looking for is Controls =